src/screens/ProposalDetail.js

```diff
@@ -360,17 +364,16 @@ function ProposalDetail({
           }
         />
       </div>
-      {/* <SidePanel
-        title="Support this proposal"
-        opened={panelState.visible}
-        onClose={panelState.requestClose}
+      <MultiModal
+        visible={miltiModalVisible}
+        onClose={() => setMiltiModalVisible(false)}
       >
-        <SupportProposalPanel
+        <SupportProposalScreens
           id={id}
-          onDone={panelState.requestClose}
-          onStakeToProposal={convictionActions.stakeToProposal}
+          onDone={() => setMiltiModalVisible(false)}
+          onStakeToProposal={actions.stakeToProposal}
```

Comment:
@rperez89 heres is where i need your help, at the moment of calling onStakeToProposal fails because onStakeToProposal is not a function.

Reply:
STEP 1
Another thing that you need to do is change the way that the stakeToProposal makes the contract call since now we have a modal transition that keeps loading while the trx is being mined as opposed to just propmpt metamask to sign as we were doing before, see here: https://github.com/1Hive/honey-pot/blob/3185f4b1c3abfb07cf89b3da4fc1ba64dbb709b6/src/hooks/useActions.js#L82-L92 there you can see that the `sendIntent` function ask ethers to get the signer what is the action that prompts metamask.
Now the modal is the one in charge of executing transactions https://github.com/1Hive/honey-pot/blob/disputable-version/src/components/ModalFlows/ModalFlowBase.js#L30-L74.
So we need to adapt the stakeToProposal action to do not call metamask and just generate an intent, an intent is kind of a pre generation of the data that needs to be send to the signer with the `from, to, data` that is going to be sent to the smart contract and you can see an example of how to do this here https://github.com/1Hive/honey-pot/blob/disputable-version/src/hooks/useActions.js#L172-L190